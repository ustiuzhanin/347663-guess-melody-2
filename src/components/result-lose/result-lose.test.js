import React from "react";
import renderer from "react-test-renderer";
import {BrowserRouter} from "react-router-dom";
import ResultLose from "./result-lose.jsx";

test(`ResultLose's snapshot`, () => {
  const tree = renderer
    .create(
        <BrowserRouter>
          <ResultLose onClick={jest.fn()} time={100} />
        </BrowserRouter>
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
