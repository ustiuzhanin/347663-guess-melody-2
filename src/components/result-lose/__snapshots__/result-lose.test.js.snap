// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResultLose's snapshot 1`] = `
<section
  className="result"
>
  <div
    className="result__logo"
  >
    <img
      alt="Угадай мелодию"
      height="83"
      src="img/melody-logo.png"
      width="186"
    />
  </div>
  <h2
    className="result__title"
  >
    Game Over!
  </h2>
  <p
    className="result__total result__total--fail"
  >
    You have exceeded the number of allowed mistakes!
    <br />
    No worries, next time is gonna be better!
  </p>
  <a
    className="replay"
    href="/"
    onClick={[Function]}
  >
    Try Again!
  </a>
</section>
`;
